### ISSUE TITLE
Gameinfo Bypass Offset is NULL!

### ISSUE THREAD

**Issue by mediv0** [Wolf49406/Dota2Patcher#19]
After todays update Patching game info is not possible

```
[+] Dota Plus Unlocked
[+] Sv_cheats Patched
[-] Gameinfo Bypass Offset is NULL!
```

**Comment by ryouik1** [comment]
+1

**Comment by Wolf49406** [comment]
So that means it already patched and you don't need to patch it again.
![image](https://user-images.githubusercontent.com/26302888/166449932-6952bdd6-9d6e-40c4-9b22-b1e7a9cb474d.png)

**Comment by Laykeel** [comment]
I suppose my game search is banned now because of no blind mod (it's still working) as I've reverted patches. 
But maybe u can tell how to unistall it for now? Cannot find instructions on its page

**Comment by Wolf49406** [comment]
@Laykeel are you retarded? Latest dota patch broke GameInfo patch signature.
And i have already fix it in 9e7e717

**Comment by Laykeel** [comment]
@Wolf49406 just have no idea how it works. Thx for patch though)
